### PR TITLE
Update fix some responsive design things

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,25 +19,25 @@ <h1>Daniel Eicher</h1>
...
   </section>
-  <section id="contact">
-    <h2>Contact</h2>
-    <ul>
-      <li class="nowrapping"><span class="bold">Phone: </span>+34642104722</li>
-      <li><span class="bold">Mail: </span><a href="mailto:danieleicher@gmx.net">danieleicher@gmx.net</a></li>
+  <section>
+    <h2 id="contact">Contact</h2>
+    <ul class="inline-flex">
+      <li><span class="bold">Phone: </span><br>+34642104722</li>
```
Comment (Quaquaro) [inline index.html:39]: for better readability you could divide the number into blocks of 3. for example +34 642 047 22

File: index.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h3>Tech stack</h3>
             src="https://upload.wikimedia.org/wikipedia/commons/c/c3/Python-logo-notext.svg" width="32" height="32"
             alt="python-logo"></a>
         <!-- Kotlin-logo with a link to kotlinlang.org -->
-        <a id="kotlin" class="logo" href="https://kotlinlang.org/" target="_blank"><img
+        <a id="kotlin-logo" class="logo" href="https://kotlinlang.org/" target="_blank"><img
```
Comment (Quaquaro) [inline index.html:79]: Cool do you write your own apps for Android?

**Review by Quaquaro — COMMENTED** [review]
That looks pretty damn good.

File: index.html (hunk 2 above)
Comment (codingoak) [reply index.html:79]: I tried 2 years ago, with some android and kotlin books and the fantastic android studio software. But i think it is a very difficult start into the world of development. I retry it with more experience.

File: index.html (hunk 1 above)
Comment (codingoak) [reply index.html:39]: Yes i changed it now, thank you

**Pull request merged**